third_party/googletest/BUILD.gn

```diff
@@ -138,13 +138,13 @@ source_set("gtest") {
   defines = []
   deps = []
   public_deps = []
-  if (is_nacl || !build_with_chromium) {
+  if (is_nacl || !build_with_chromium || is_starboard) {
     defines += [ "GTEST_DISABLE_PRINT_STACK_TRACE" ]
     sources -= [
       "custom/gtest/internal/custom/stack_trace_getter.cc",
       "custom/gtest/internal/custom/stack_trace_getter.h",
     ]
-  } else {
+  } else if (!is_starboard) {
```

Comment:
This can just be kept as `else` with `|| is_starboard` added to the above condition